xml/System.Text.Json.Serialization.Metadata/JsonCollectionInfoValues`1.xml

```diff
@@ -26,7 +26,7 @@
   <Docs>
     <typeparam name="TCollection">The collection type.</typeparam>
     <summary>Provides serialization metadata about a collection type.</summary>
-    <remarks>This API is intended for use by the output of the System.Text.Json source generator and should not be called directly.</remarks>
```

Comment:
This remark still holds -- this type is marked EditorBrowsable.Never and is only intended for consumption by the source generator.